src/plone/volto/profiles/default/catalog.xml

```diff
@@ -3,4 +3,5 @@
   <column value="nav_title" />
   <column value="head_title" />
   <column value="hasPreviewImage"/>
+  <column value="image_field"/>
```

Comment:
@sneridagh @ericof I am wondering here if "preview_image_field" or "teaser_image_field" (if we want to make teaser a first-class citizen, we already have the new teaser image scale in plone.volto) isn't a more specific name.

Reply:
Note: discussed this in a call. We keep the name because this points to the image_field which does not necessarily mean the preview_image.